content/blog/2019/02/2019-02-17-remoting-cli-removed.adoc

```diff
@@ -9,7 +9,7 @@ author: jglick
 ---
 
 Close to two years ago, we announced in
-link:../../2017/04/2017-04-11-new-cli/[New, safer CLI in 2.54]
+link:/blog/2017/04/11/new-cli/[New, safer CLI in 2.54]
```

Comment:
@darxriggs alerted me to 404s. Seems that blog posts have a URL structure that you would not predict by their source file location. (Why?)

Also switching to server-absolute URLs, probably clearer.